src/ocamlorg_web/lib/templates/previews/dune

```diff
@@ -0,0 +1,77 @@
+(rule
```

Comment:
Minor (feel free to ignore): I think we can avoid all of the duplication here with a single rule

```
(rule
 (targets preview_books_template.ml preview_papers_template.ml ...)
 (deps preview_books_template.eml preview_papers_template.ml ...)
 (action
  (run %{bin:dream_eml} %{deps} --workspace %{workspace_root})))
```

Just dawned on me whilst reviewing.

Reply:
Unfortunately, the Dream CLI does not accept multiple files as input, so we have to run one action for each of the files.

Reply:
I think it does https://github.com/aantron/dream/blob/master/src/eml/main.ml#L40 ? It works locally for me (`dream.1.0.0~alpha2`). I think the `--help` says `FILE` when it should probably say `FILES`